templates/.circleci/config.yml.tpl

```
@@ -61,9 +61,22 @@ workflows:
       ###EndBlock(circleWorkflowJobs)
       {{- if $testNodeClient }}
       - shared/test_node_client:
+          context: *contexts
+          steps:
+            ###Block(testNodeClientSteps)
+            {{- if empty (file.Block "testNodeClientSteps") }}
+            []
+            {{- else }}
+{{ file.Block "testNodeClientSteps" | fromYaml | toYaml | indent 12 }}
+            {{- end }}
+            ###EndBlock(testNodeClientSteps)
           requires:
             ###Block(testNodeRequires)
+            {{- if empty (file.Block "testNodeRequires") }}
+            []
+            {{- else }}
 {{ file.Block "testNodeRequires" | fromYaml | toYaml | indent 12 }}
+            {{- end }}
```

Might be able to shorten this to:

```suggestion
{{ file.Block "testNodeRequires" | default "[]" | fromYaml | toYaml | indent 12 }}
```